### PR TITLE
Code and interface improvements

### PR DESCRIPTION
**Pull request by wes1350**
Add docstrings, type annotations, command line functionality, improve prompts and messages

### CODE REVIEW

File: classes/characters/Ambassador.py [hunk 1]
```diff
@@ -5,5 +5,8 @@ class Ambassador(Character):
     def __init__(self):
         pass        
     
+    def description(self) -> str:
+        return "Ambassadors can do the following:\n    Exchange: Exchange cards with the Court. First take 2 random cards from the Court deck. Choose which, if any, to exchange with your face-down cards. Then return two cards to the Court deck. \n    Block: Steal"
```
Comment (tzerchyuan) [inline classes/characters/Ambassador.py:9]: instead of 2, use the dynamic value from config

Comment (tzerchyuan) [reply classes/characters/Ambassador.py:9]: but nbd

**Comment by tzerchyuan** [comment]
Merging is up to the author! Good shit

File: classes/characters/Ambassador.py (hunk 1 above)
Comment (wes1350) [reply classes/characters/Ambassador.py:9]: ah yes we should include that, i copy/pasted from the coup site

**Pull request merged**